lib/new_relic/agent/instrumentation/grpc/client/instrumentation.rb

```diff
@@ -73,10 +70,16 @@ def trace_with_newrelic?(host = nil)
             return do_trace unless do_trace.nil?
 
             host ||= @host
-            return false unless host && !host.match?(HOST_BLOCKLIST_PATTERN)
+            return false unless host && !host_denylisted(host)
 
             true
           end
+
+          def host_denylisted(host)
+            NewRelic::Agent.config[:'instrumentation.grpc.host_denylist'].any? do |regex|
+              host.match(regex)
```

Comment:
I think `#match?` would be better than `#match` here. Giving `#any?` a simple boolean seems better than a match object, given that we don't actually perform any captures or anything that the match object would be used for.

Reply:
resolved: b05ee085cfd6e726be0285827c325be5a5aca9cd